src/components/image/index.test.tsx

```diff
@@ -23,7 +23,7 @@ import React from "react";
 import { render, waitFor } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
 
-import * as SolidFns from "@inrupt/solid-client";
+import SolidFns from "@inrupt/solid-client";
```

Comment:
This change looks weird, because you should definitely be able to do `import * as solidFns from "@inrupt/solid-client"` though, better would be to import the specific functions desired here.

Reply:
The import itself is fine, but the `jest.spyOn` complains about it at runtime.

Reply:
so what, `jest.spyOn` is on the `SolidFns` instead of the method? I'd like to better understand what's going on here.

Reply:
@ThisIsMissEm I do not quite understand why this error was happening either, but essentially we were getting these errors: 
![image](https://user-images.githubusercontent.com/28412960/215445179-f0e41b8d-6dfd-44c9-93e5-08d58db12a5c.png)

Reply:
Likely because they are exported as `const` would be my guess. That or some other sort of weird freezing behaviour with exports.